templates/nginx.conf.nj

```diff
@@ -41,7 +41,16 @@ http {
     rewrite ^/{{ page.from }}(.*)$ https://{{ page.to }}.18f.gov$1;
     {% endfor %}
 
+    rewrite ^/intake/.* https://18f.gsa.gov/contact/ redirect;
+    rewrite ^/designstandards/.* https://standards.usa.gov/ redirect;
+    rewrite ^/pages/.* https://github.com/18F/pages/ redirect;
```

Comment:
@wslack This one seems weird to me. Basically it redirects https://pages.18f.gov/pages/whatever` back to the pages GitHub repo. Is it necessary to keep?

Copied from https://github.com/18F/federalist-redirects/blob/master/nginx.conf#L87